### ISSUE TITLE
the new pipeline view is hawt; would be great to fold/hide the completed builds somehow

### ISSUE THREAD

**Issue by christian-posta** [fabric8io/fabric8#5017]
in the pipeline, so that it doesn't wrap across the screen..
like an accordion or something, it would hide, or minimize the completed/successful builds given the real-estate of the screen so we can see which build is in progress easier w/out following arrows across the screen.

**Comment by rawlingsj** [comment]
Yeah almost like a conveyor belt.  As the pipeline progresses its good to know what the previous, current and next stages are but it would also be nice to be able to scroll back and forth.
